packages/reactivity/src/reactive.ts

```diff
@@ -8,6 +8,7 @@ import {
 
 import { UnwrapNestedRefs } from './ref'
 import { ReactiveEffect } from './effect'
+import { vnodeSymbol } from '@vue/runtime-core'
```

Comment:
This cross package import breaks the dependency graph because the reactivity package needs to be usable standalone.

Exporting `vnodeSymbol` either from `runtime-core` or a separate package both essentially allows the user to use it to fake a vnode, defeating the purpose. So I don't think it's worthwhile for VNodes.

Reply:
I think it's not a protection of intentional faking but just a feature so user won't fake VNode accidentally. But we also can keep symbols package private.